support/Sources/GraphQL.swift

```diff
@@ -231,8 +231,8 @@ public class GraphQL {
 }
 
 public struct GraphQLResponse<DataType: GraphQL.AbstractResponse> {
-	public let data: DataType?
-	public let errors: [GraphQL.ResponseError]?
+	internal let data: DataType?
+	internal let errors: [GraphQL.ResponseError]?
```

Comment:
these should be left public, they are the purpose of this struct (deserializing response into errors/data)

Reply:
They aren't needed on any of the generated models.

Reply:
They are/can be used by external code though. From the README

```
guard let response = try? GraphQLResponse<ExampleSchema.QueryRoot>(jsonData: response) else {
    print("Invalid GraphQL response")
    return
}
if let errors = response.errors {
    for error in errors {
        print("GraphQL error: \(error.message)")
    }
}
if let data = response.data {
    let user = data.user
    print("\(user.firstName) \(user.lastName)")
}
```

Reply:
What is `GraphQLResponse` used for? I can't find any usage of it. Can we not just remove it entirely?